sdc/hiframes/api.py

```diff
@@ -162,7 +163,7 @@ def fix_df_array_list_str_impl(column):  # pragma: no cover
     if isinstance(column, RangeIndexType):
         return lambda column: np.array(column)
 
-    if isinstance(column, (types.Array, StringArrayType)):
+    if isinstance(column, (types.Array, StringArrayType, Categorical)):
```

Comment:
Are there other methods which need the same fix?

Reply:
I think half of all DF and Series methods should have some special handling of Categoricals (see e.g. df.values). If we talk only about boxing/unboxing/fix_df_array/fix_df_index function, then probably no, since we don't support CategoricalIndexes yet. At least it looks so.